.ci/appveyor.yml

```diff
@@ -49,6 +57,26 @@ install:
   - "npm config set loglevel warn"
   - "npm install"
 
+  - "curl -sSf -o rustup-init.exe https://win.rustup.rs/"
+  - "rustup-init.exe -y --default-toolchain=%RUST_TOOLCHAIN%"
+  # Normally, we shouldn't need the toolchain lib directory in our PATH
+  # https://github.com/rust-lang-nursery/rustup.rs/issues/876
+  # This is actually the sole reason that we really
+  # need the RUST_TOOLCHAIN variable.
+  - "set PATH=%PATH%;C:\\Users\\appveyor\\.cargo\\bin"
+  # Start ignoring YAMLLintBear
+  # because of the line length; plitting up these paths would become unreadable
+  - "set PATH=%PATH%;C:\\Users\\appveyor\\.rustup\\toolchains\\%RUST_TOOLCHAIN%\\bin"
+  - "set PATH=%PATH%;C:\\Users\\appveyor\\.rustup\\toolchains\\%RUST_TOOLCHAIN%\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib"
+  # Stop ignoring YAMLLintBear
+  - "cargo install clippy --vers 0.0.144"
+
+  # Print rust and cargo parameters for quick debugging.
```

Comment:
I pot it there because it might get in handy in the future, when things start to break on the CI because of me :'-)